src/main/kotlin/to/rxs/kommunity/Config.kt

```diff
@@ -31,3 +33,13 @@ enum class Environment {
     DEVELOPMENT,
     PRODUCTION
 }
+
+fun createRolesMap  (env: String) : Map<String, Snowflake> {
```

Comment:
```suggestion
fun createRolesMap(env: String) : Map<String, Snowflake> {
```

This kills my eyes I am blind now